lib/chef/knife/core/generic_presenter.rb

```diff
@@ -175,23 +175,19 @@ def formatting_subset_of_data?
 
         def extract_nested_value(data, nested_value_spec)
           nested_value_spec.split(".").each do |attr|
-            if data.nil?
-              nil # don't get no method error on nil
-              # Must check :[] before attr because spec can include
-              #   `keys` - want the key named `keys`, not a list of
-              #   available keys.
-            elsif data.respond_to?(:[]) && data.has_key?(attr)
-              data = data[attr]
-            elsif data.respond_to?(attr.to_sym)
-              data = data.send(attr.to_sym)
-            else
-              data = begin
-                       data.send(attr.to_sym)
-                     rescue NoMethodError
-                       nil
-                     end
-            end
+            data =
```

Comment:
Also this would only work with nodes, not sure if `-a` exists on any non-node data sources. Probably does, just to make things difficult.